packages/tenant-process/test/integration/getTenantVerifiedAttributeRevokers.test.ts

```diff
@@ -0,0 +1,286 @@
+import {
+  getMockAttribute,
+  getMockContext,
+  getMockTenant,
+} from "pagopa-interop-commons-test";
+import {
+  addOneAttribute,
+  addOneTenant,
+  tenantService,
+} from "../integrationUtils.js";
+import {
+  getMockVerifiedTenantAttribute,
+  getMockCertifiedTenantAttribute,
+} from "../mockUtils.js";
+
+describe("getTenantVerifiedAttributeRevokers", () => {
+  const tenantId: TenantId = generateId();
+  const attributeId: AttributeId = generateId();
+  const revoker1Id: TenantId = generateId();
+  const revoker2Id: TenantId = generateId();
+
+  const revoker1: TenantRevoker = {
+    id: revoker1Id,
+    verificationDate: new Date("2024-01-01T10:00:00Z"),
+    revocationDate: new Date("2024-06-01T10:00:00Z"),
+    expirationDate: undefined,
+    extensionDate: undefined,
+  };
+
+  const revoker2: TenantRevoker = {
+    id: revoker2Id,
+    verificationDate: new Date("2024-02-01T10:00:00Z"),
+    revocationDate: new Date("2024-07-01T10:00:00Z"),
+    expirationDate: undefined,
+    extensionDate: undefined,
+  };
+
+  const verifiedAttribute: VerifiedTenantAttribute = {
+    ...getMockVerifiedTenantAttribute(),
+    id: attributeId,
+    assignmentTimestamp: new Date("2024-01-01"),
+    verifiedBy: [],
+    revokedBy: [],
+  };
+
+  const tenant: Tenant = {
+    ...getMockTenant(),
+    id: tenantId,
+    attributes: [verifiedAttribute],
+  };
+
+  beforeEach(async () => {
+    await addOneTenant({
+      ...getMockTenant(),
+      id: revoker1Id,
+    });
+    await addOneTenant({
+      ...getMockTenant(),
+      id: revoker2Id,
+    });
+    await addOneAttribute({
+      ...getMockAttribute(attributeKind.verified),
+      id: attributeId,
+    });
+  });
+
+  it("should return revokers with correct details", async () => {
+    const tenantWithRevokers: Tenant = {
+      ...getMockTenant(),
+      id: generateId(),
+      attributes: [
+        {
+          ...getMockVerifiedTenantAttribute(),
+          id: attributeId,
+          assignmentTimestamp: new Date("2024-01-01"),
+          verifiedBy: [],
+          revokedBy: [revoker1, revoker2],
+        },
+      ],
+    };
+
+    await addOneTenant(tenantWithRevokers);
+
+    const result = await tenantService.getTenantVerifiedAttributeRevokers(
+      tenantWithRevokers.id,
+      attributeId,
+      { offset: 0, limit: 10 },
+      getMockContext({})
+    );
+
+    expect(result.results).toHaveLength(2);
+
+    const resultIds = result.results.map((revoker) => revoker.id);
+    expect(resultIds).toContain(revoker1.id);
+    expect(resultIds).toContain(revoker2.id);
+
+    // Check that we get full TenantRevoker objects with all required fields
+    result.results.forEach((revoker) => {
+      expect(revoker).toHaveProperty("id");
+      expect(revoker).toHaveProperty("verificationDate");
+      expect(revoker).toHaveProperty("revocationDate");
+      expect(revoker.verificationDate).toBeInstanceOf(Date);
+      expect(revoker.revocationDate).toBeInstanceOf(Date);
+    });
```

Comment:
Here in this main test, let's hardcode an explicit check of the entire results object, as we do in other similar tests, so that we are forced to update the test in case we add new properties in the future or some value conversion changes

```suggestion
    result.toEqual({
        totalCount: 2,
        results: [revoker1, revoker2] // maybe some conversion is needed here....
    })
```